File: backend/src/main/java/com/java/NBE4_5_3_7/Nbe4537Application.java
package com.java.NBE4_5_3_7;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableJpaAuditing
@EnableScheduling
public class Nbe4537Application {

	public static void main(String[] args) {
		SpringApplication.run(Nbe4537Application.class, args);
	}

}

File: backend/src/main/java/com/java/NBE4_5_3_7/Nbe4537Application.kt
package com.java.NBE4_5_3_7;

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cache.annotation.EnableCaching
import org.springframework.data.jpa.repository.config.EnableJpaAuditing
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableJpaAuditing
@EnableScheduling
@EnableCaching
class Nbe4537Application

fun main(args: Array<String>) {
	runApplication<Nbe4537Application>(*args)
}